EIPS/eip-4844.md

```diff
@@ -234,6 +232,44 @@ def get_origin(tx: SignedBlobTransaction) -> Address:
     return ecrecover(tx_hash(tx), int(sig.y_parity)+27, sig.r, sig.s)
 ```
 
+### Header extension
+
+The current header encoding is extended with a new 256-bit unsigned integer field `blob_txs`. This is the running
+total of blob txs included on chain since this EIP was activated. The resulting RLP encoding of the header is therefore:
+
+```
+rlp([
+    parent_hash,
+    ommers_hash,
+    coinbase,
+    state_root,
+    tx_root,
+    receipt_root,
+    bloom,
+    difficulty,
+    number,
+    gas_limit,
+    gas_used,
+    time,
+    extra,
+    mix_digest,
+    nonce,
+    base_fee,
+    blob_txs
+])
+```
+
+The value of `blob_txs` can be calculated using the parent header and number of blob transactions in the block.
+
+```python
+def calc_blob_gas(parent: Header, blob_txs_in_block: int) -> int:
```

Comment:
Rename to `calc_blob_txs`?

Reply:
On second thoughts, a potential even better idea: instead of storing a "total" of `blob_txs`, which is not actually even a total because of how it gets topped up if it gets too low, why not store `excess_blob_txs`?

```python
def calc_excess_blob_txs(parent: Header, blob_txs_in_block: int) -> int:
    output = parent.blob_txs + blob_txs_in_block
    output = output - min(TARGET_BLOB_TXS_PER_BLOCK, output)
```

Reply:
I've updated the PR to use this technique, PTAL!